windpowerlib/modelchain.py

```diff
@@ -438,6 +439,11 @@ def run_model(self, weather_df):
         'wind_speed'
 
         """
+        # Convert data heights to integer. In some case they are strings.
+        weather_df.columns = pd.MultiIndex.from_arrays([
+            weather_df.columns.get_level_values(0),
+            weather_df.columns.get_level_values(1).astype(int)])
```

Comment:
Maybe putting in a `try` statement and re-raising the error? In case someone passes in  `80 m` that cannot be converted to a number?

Reply:
For me the error message is quite clear. It also indicates the exact value that causes the problem. For us it would be more difficult to indicate the problematic value. We could add something like "The heights should be convertible to numeric." but it in my opinion the original message is better. We could also check the array and show all problematic values but I think it is not worth it.